site/packages/aa-alchemy/signer/authenticate.md

```diff
@@ -0,0 +1,69 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: Alchemy Signer • authenticate
+  - - meta
+    - name: description
+      content: Learn how to use the AlchemySigner.authenticate method
+  - - meta
+    - property: og:description
+      content: Learn how to use the AlchemySigner.authenticate method
+  - - meta
+    - name: twitter:title
+      content: Alchemy Signer • authenticate
+  - - meta
+    - name: twitter:description
+      content: Learn how to use the AlchemySigner.authenticate method
+---
+
+# authenticate
+
+The `authenticate` method is used to authenticate a user with the Alchemy Signer.
+
+## Usage
+
+::: code-group
+
+```ts
+import { signer } from "./signer";
+
+const bundlePromise = new Promise(async (resolve) => {
+  const otpFromUser = await getOtpFromUser();
+  resolve(otpFromUser);
+});
+
+const user = await signer.authenticate({
+  type: "email",
+  email: "user@email.com",
+  bundle: bundlePromise,
```

Comment:
what is a bundle exactly?

Reply:
bundle is the OTP string that a user is emailed. we'll be killing this in the next week or two anyways so likely won't add much more than that for description